configmaptocrs/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
```

Comment:
should you build here instead so we won't need to revisit it later? https://github.com/metallb/metallb/issues/1408

Reply:
That will require implementing #1408 here (partially) because of how inv build works, so I'd defer it.